src/Teams/Services/ManageSprintsService.cs

```diff
@@ -37,5 +39,14 @@ public async Task<Team> GetTeam(int team_id)
         }
 
         public async Task<Sprint> GetSprintAsync(int sprint_id) => await _sprintRepository.GetByIdAsync(sprint_id);
+
+        public async Task<bool> AddSprintAsync(Sprint sprint)
+        {
+            if (_sprintRepository.GetAll().Where(x=>x.TeamId == sprint.TeamId).Any(x=>x.Name == sprint.Name) || sprint.DaysInSprint<=0 || sprint.StorePointInHours<=0 || !Regex.IsMatch(sprint.Name, ("^[a-zA-Z0-9-_.]+$")))
```

Comment:
if one line too long divide it into many lines like this:
_sprintRepository.GetAll()
                            .Where . . . 
                            .Any . . .